lib/trino-parquet/src/main/java/io/trino/parquet/writer/ParquetSchemaConverter.java

```diff
@@ -95,6 +103,25 @@ public MessageType getMessageType()
         return messageType;
     }
 
+    private MessageType toMessageType(
```

Comment:
let's use `convert` as name of the method - to be in sync with the existing methods.

Reply:
ok. fixed.